### PR TITLE
Forms, incl. Board's create item flow and a11y improvements

### PR DESCRIPTION
**Pull request by thure**
## Specs in Figma
• [Fluent UI forms ↗️](https://www.figma.com/file/1B2PfJElNaCydW5SKeQOBK/Form-Fluent-UI?node-id=0%3A1)
• [Forms in MS Teams UI design kit ↗️](https://www.figma.com/file/UwN095fVIvwZDNtzTtUE3E/Microsoft-Teams-UI-Design-Kit?node-id=1%3A34)
• [Board in “App Samples” ↗️](https://www.figma.com/file/gFdfpdTybsG5thYwSiGsI8/App-samples?node-id=0%3A1)

### CODE REVIEW

File: src/themes/shared/components/Dropdown/dropdown-styles.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+> = {
+  root: (componentStyleParameters) => {
+    const {
+      props,
+      theme: {
+        siteVariables: { colorScheme },
+      },
+    } = componentStyleParameters;
+    return {
+      borderRadius: "4px",
```
Comment (akatrukhin) [inline src/themes/shared/components/Dropdown/dropdown-styles.ts:25]: Should we use border radius from siteVariables?

File: src/themes/shared/static-styles.ts [hunk 2]
```diff
@@ -4,11 +4,21 @@
...
+  `${focusableSelectors
+    .map(
+      (selector) =>
+        `html[data-whatinput="keyboard"] ${selector}::before, html[data-whatinput="keyboard"] ${selector}::after`
+    )
+    .join(", ")} {
     border-width: 2px !important;
-  }`,
-  `html[data-whatinput="keyboard"] :focus::after, html[data-whatinput="keyboard"] :focus::before {
     border-radius: 4px !important;
```
Comment (akatrukhin) [inline src/themes/shared/static-styles.ts:22]: if it's common and it's a part of Shell 2.0 styles, should be declared there and overwrited default borderRadius across the project?

**Review by akatrukhin — APPROVED** [review]

**Review by vikramjit-singh-chug — COMMENTED** [review]
There is an issue with alignment on mobile: @thure 

MicrosoftTeams-image (3)

MicrosoftTeams-image (2)

File: src/themes/shared/static-styles.ts (hunk 2 above)
Comment (thure) [reply src/themes/shared/static-styles.ts:22]: `themes/shared` is where many of the Shell 2.0 styles are declared, so I think this is the best place for this.

File: src/themes/shared/components/Dropdown/dropdown-styles.ts (hunk 1 above)
Comment (thure) [reply src/themes/shared/components/Dropdown/dropdown-styles.ts:25]: It's an excellent question, I'll check. I don't think `siteVariables` declares any `borderRadius`, but maybe it's high time it does.

**Comment by thure** [comment]
Screen Shot 2020-12-28 at 20 03 45

@vikramjit-singh-chug I've resolved input alignment issues; accessibility has remained the same. Could you please re-review this PR and approve?

**Review by vikramjit-singh-chug — APPROVED** [review]

**Review by asnow003 — APPROVED** [review]

**Pull request merged**